uxarray/grid/grid.py

```diff
@@ -375,14 +375,13 @@ def Mesh2_node_cart_x(self) -> xr.DataArray:
     @property
     def Mesh2_face_x(self) -> xr.DataArray:
         """UGRID Coordinate Variable ``Mesh2_face_x``, which contains the
-        longitude of each face center.
+        longitude or cartesian coordinate of each face center.
```

Comment:
`Mesh2_face_x` can only contain the longitude of each face center, this seems to be left-over from a previous version of this PR. we can revert back to the original docstring.